cmd/security-secretstore-setup/res/configuration.yaml

```diff
@@ -38,6 +38,8 @@ SecretStore:
   PasswordProviderArgs: []
   RevokeRootTokens: true
   ConsulSecretsAdminTokenPath: /tmp/edgex/secrets/edgex-consul/admin/token.json
+Database:
+  Type: "postgres"
```

Comment:
move `Type` under the exsting `Database` section, and the default value is still redisdb

Reply:
Done